nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/JdbcCommon.java

```diff
@@ -279,7 +280,7 @@ public static Schema createSchema(final ResultSet rs, String recordName, boolean
                     break;
 
                 case INTEGER:
-                    if (meta.isSigned(i)) {
+                    if (meta.isSigned(i) || (meta.getPrecision(i) > 0 && meta.getPrecision(i) <= MAX_DIGITS_IN_INT)) {
```

Comment:
I'm not sure if the meta.getPrecision(i) > 0 check should be >=, or if it is needed at all. I can't find a type on any of my databases that returns 0 (or negative) for precision for INTEGER types